eng/scripts/Language-Settings.ps1

```diff
@@ -101,7 +101,7 @@ function Publish-python-GithubIODocs ()
     Write-Host "Discovered Package Name: $PkgName"
     Write-Host "Discovered Package Version: $Version"
     Write-Host "Directory for Upload: $UnzippedDocumentationPath"
-
-    Upload-Blobs -DocDir $UnzippedDocumentationPath -PkgName $PkgName -DocVersion $Version
```

Comment:
Need to also update PackageInfoFromPackageFile  function to add ReleaseTag

Reply:
I'll do that when am restructuring that `artifact-metadata-parsing` file. right now am just working on blobs. I think I can get that completed with template releases.